example/sparse/wide_deep/inference.py

```diff
@@ -0,0 +1,121 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Can we add python3 shebang?

Reply:
Hi Larroy, thanks for review, I think this script can work with Python2, it might be fine without python3 shebang?